x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.scss

```
@@ -26,7 +26,7 @@
     background: $euiColorEmptyShade;
     height: 100%;
 
-    > * {
+    >* {
```

Small formatting suggestion.

```suggestion
    & > * {
```